README.md

```diff
@@ -1,2 +1,85 @@
-# webpack
-This is a webpack boilerplate
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# To-Do List
+
+> In this project, I built a simple HTML list of To Do tasks. The list is styled according to the specifications listed in the lesson. This simple web page is built using webpack and served by a webpack dev server.
+
+## Built With
+
+  *Major languages:*
+
+- HTML
+- CSS
+- JavaScript
+- Webpack
+- Webpack dev server
+
+## Live Demo
+
+[To-Do list structure](https://krisbobo.github.io/awesomebooks-ES6/)
```

Comment:
[OPTIONAL] It would be better to update the demo live link with the proper one so your Readme is professional.